openprescribing/frontend/management/commands/scrape_numbers.py

```diff
@@ -0,0 +1,243 @@
+# coding=utf8
```

Comment:
I'm guessing this was an old name for this file and was committed by accident

Reply:
It's because the file contains a non-ascii `£`.  Without this line, Python raises `SyntaxError: Non-ASCII character '\xc2' in file...`.